Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/Interactable.cs

```diff
@@ -989,18 +1131,111 @@ protected int GetVoiceCommandIndex(string command)
 
         #endregion VoiceCommands
 
+        #region NearInteractionHandlers
+        /// <summary>
+        /// Is the eventData a grip action or near pointer?
+        /// </summary>
+        /// <param name="eventData"></param>
+        /// <returns></returns>
+        protected bool IsGrip(IMixedRealityPointer pointer, MixedRealityInputAction action)
+        {
+            if (action.Description.ToLower().IndexOf("grip") > -1 || pointer as IMixedRealityNearPointer != null)
+            {
+                return true;
+            }
+
+            return false;
+        }
+
         void IMixedRealityTouchHandler.OnTouchStarted(HandTrackingInputEventData eventData)
         {
             SetPress(true);
```

Comment:
It's curious that both press and physical touch are set to true here. It seems like we should distinguish press from physical touch, or document that the two are the same.

Reply:
When the PhysicalPressEventRouter was created, some of this code was added, so I tried to make sure I wasn't breaking any existing patterns. What I got from it was Press was like a fallback state if we weren't supplying feedback for physical touch. So at least something was happen on contact even thought the button doesn't move. Documentation here would help.

Reply:
I'll update the documentation. Press is a fallback to Physical Press, so we get some feedback if physical press is not being monitored.